client/luigi-client.js

```diff
@@ -169,8 +169,8 @@
       return id;
     },
     /**
-     * Removes a init listener
-     * @param {string} id the id that was returned by `addInitListener`
+     * Removes the init listener.
```

Comment:
There can be multiple init listeners, the function removes the listener based on the id. 
The @param change is fine, `the` is ok here.

Reply:
Thank you, updated.